src/django/api/views/moderation_events/merge_events.py

```diff
@@ -62,30 +59,21 @@ def generate_detail_fields(detail: bool) -> str:
                        "ahf.address AS original_address") or ""
 
 
-def generate_source_data_sql(all: bool, contributors: list,
-                             contributor_id: int) -> str:
-    if all:
+def generate_source_data_sql(contributors: list) -> str:
+    if not contributors:
+
         return (" FROM api_historicalfacility ahf "
                 "JOIN api_facilityalias afa ON afa.os_id = ahf.id ")
 
     if contributors:
```

Comment:
You can remove this `if` clause since you already have one check(`if not contributors`) above. The return statement below will serve as the default.

Reply:
removed